doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -32,21 +32,25 @@ datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_files = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)
 
 They return a data structure that contains different pieces of
 information on the retrieved dataset, including the
 file names on hard disk::
 
     >>> # The different files
-    >>> print(sorted(list(haxby_files.keys())))  # doctest: +SKIP
-    ['anat', 'description', 'func', 'mask_face', 'mask_face_little', 'mask_house', 'mask_house_little', 'mask_vt', 'session_target']
+    >>> print(sorted(list(haxby_dataset.keys()))) # doctest: +SKIP
+    ['anat', 'description', 'func', 'mask_face', 'mask_face_little',
+    'mask_house', 'mask_house_little', 'mask_vt', 'session_target']
     >>> # Path to first functional file
-    >>> print(haxby_files.func[0])  # doctest: +ELLIPSIS +SKIP
+    >>> print(haxby_dataset.func[0]) # doctest: +ELLIPSIS
```

Comment:
Same remark about skipping the doctest. It's probably a bad idea not to skip them, as it will entail heavy downloads on travis.

Another option would be to mock or modify dataset downloads while testing. I'm open to it, but it requires much more work.